src/features/notes/notesCategory/notesCategorySlice.js

```diff
@@ -56,12 +57,56 @@ export const deleteNotesCategory = createAsyncThunk("notesCategory/delete", asyn
         return thunkAPI.rejectWithValue(message);
     }
 });
+// IsSelected user category: note
+export const AddSelectedCategory = createAsyncThunk("notesCategory/selected", async (id, thunkAPI) => {
```

Comment:
```suggestion
export const addSelectedCategory = createAsyncThunk("notesCategory/selected", async (id, thunkAPI) => {
```


I didn't understand why you use createAsyncThunk for a non async function